packages/point-of-sale/src/graphql/queries/getWalletAddress.ts

```diff
@@ -0,0 +1,53 @@
+import { gql } from '@apollo/client'
+
+export const GET_WALLET_ADDRESS_BY_URL = gql`
+  query GetWalletAddress(
+    $url: String!
+    $first: Int
+    $last: Int
+    $before: String
+    $after: String
+    $sortOrder: SortOrder
+  ) {
+    walletAddressByUrl(url: $url) {
+      id
+      incomingPayments(
+        first: $first
+        last: $last
+        before: $before
+        after: $after
+        sortOrder: $sortOrder
+      ) {
+        edges {
+          node {
+            id
+            url
+            walletAddressId
+            client
+            state
+            incomingAmount {
+              assetCode
+              assetScale
+              value
+            }
+            receivedAmount {
+              assetCode
+              assetScale
+              value
+            }
+            expiresAt
+            createdAt
+            tenantId
```

Comment:
We actually dont have `tenantId` being returned in `paymentToGraphql` in `packages/backend/src/graphql/resolvers/incoming_payment.ts`